tests/returntypes/test_new_return_types_qnode.py

```diff
@@ -417,7 +417,7 @@ def test_sample(self, measurement, device, shots=100):
         """Test the sample measurement."""
         import tensorflow as tf
 
-        if device == "default.mixed" or "default.qubit":
+        if device in ["default.mixed", "default.qubit"]:
```

Comment:
You should remove the skip in the tests, now those tests should pass right?

Reply:
they do not pass ;(
- with `diff_method=None`, the type of the qnode output is a numpy.ndarray instead of a tf.Tensor when the device is not `default.qubit.tf` (see #2904). This is true even with the old return type iirc
- we don't yet support other diff methods afaik

I'll try to address these issues in a future PR